docs/api.rst

```diff
@@ -0,0 +1,53 @@
+.. _api:
```

Comment:
Far from perfect but we can start with it - there are basic examples that should be sufficient.